### PR TITLE
fzy 0.8 (new formula)

### PR DESCRIPTION
**Pull request by sbdchd**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
homepage: https://github.com/jhawthorn/fzy

### CODE REVIEW

File: Formula/fzy.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  head "https://github.com/jhawthorn/fzy.git"
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+    man.install "fzy.1"
+  end
+
+  test do
+    system "#{bin}/fzy", "--help"
```
Comment (bfontaine) [inline Formula/fzy.rb]: Please write a test that does something more substantial than `--help`.

File: Formula/fzy.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  desc "Fast, simple fuzzy text selector with an advanced scoring algorithm."
+  homepage "https://github.com/jhawthorn/fzy"
+  url "https://github.com/jhawthorn/fzy/archive/0.8.tar.gz"
+  sha256 "44293ef6a33ef5c9400b9546c580170a2df0ac4342b8fe447cee2af35e241b89"
+  head "https://github.com/jhawthorn/fzy.git"
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+    man.install "fzy.1"
```
Comment (bfontaine) [inline Formula/fzy.rb]: This should go in `man1`. Why doesn’t the `make install` step take care of that?

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by sbdchd** [comment]
@bfontaine I have updated the formula with a new test.

File: Formula/fzy.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  head "https://github.com/jhawthorn/fzy.git"
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    _, _, stderr = Open3.popen3("#{bin}/fzy -l 0")
+    assert_match /Must be integer in range 3/, stderr.read
```
Comment (bfontaine) [inline Formula/fzy.rb]: You can use `shell_output`:
```ruby
assert_match "Must be integer in range 3", shell_output("#{bin}/fzy -l 0")
```

Comment (sbdchd) [inline Formula/fzy.rb]: That doesn't seem to work as `fzy` is sending data via stderr

Comment (bfontaine) [inline Formula/fzy.rb]: Sorry; add `2>&1` at the end.

Comment (sbdchd) [inline Formula/fzy.rb]:
```rb
assert_match "Must be integer in range 3", shell_output("#{bin}/fzy -l 0 2>&1")
```
The above assertion doesn't work with my setup.

Is there a straightforward way to debug the comparison?

**Comment by sbdchd** [comment]
@bfontaine updated with your test suggestion

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @sbdchd for your contribution to Homebrew!

**Pull request closed without merging**